toxcore/group.c

```diff
@@ -3294,6 +3294,11 @@ static State_Load_Status load_conferences(Group_Chats *g_c, const uint8_t *data,
         }
 
         g->title_len = *data;
+
+        if (g->title_len > MAX_NAME_LENGTH) {
+            return STATE_LOAD_STATUS_ERROR;
```

Comment:
There is an allocation that happens above:

```
g->frozen = (Group_Peer *)malloc(sizeof(Group_Peer) * g->numfrozen);
```

do we need to free it before returning or is it fine to just return?

Reply:
AFAICT this causes a memory leak, but the whole function has this problem, so I'm going to address it in a separate PR.

Reply:
@zugz, your code is leaking.